### PR TITLE
fix: plain console mode

### PR DESCRIPTION
**Pull request by pavel-github**
- [x] Ready for review
- [x] Follows CONTRIBUTING rules
- [ ] Reviewed by Snyk internal team

#### What does this PR do?
This PR forces `plain` console mode when executing Gradle commands. If it's not set, then build output contains control ANSI characters and generated JSON cannot be parsed.

This PR was triggered by failed issues in teamcity plugin.

#### Where should the reviewer start?
Compare 2 Travis builds:
- failed build after console mode is manually set from outside: https://travis-ci.org/snyk/snyk-gradle-plugin/builds/633516869
- passed build after console mode is set for each Gradle command executing https://travis-ci.org/snyk/snyk-gradle-plugin/builds/633540774

### CODE REVIEW

**Comment by lili2311** [comment]
@pavel-github could you please also attach a screenshot of the difference you get today when running with CLI before and with this change? Will the user see anything different?

**Comment by pavel-github** [comment]
@lili2311, I have only screenshot with "fail-behavior" because our teamcity plugin has bundled CLI.
teamcity__build-output-with-verbose-console

**Comment by lili2311** [comment]
@pavel-github I was specifically wondering how does this affect the cli, you can test it by locally linking this branch with `npm link` and then running `npm link snyk-gradle-plugin` in the locally built cli and test the difference between the console of released cli and this

**Comment by gitphill** [comment]
paired with @pavel-github to show how to generate the screen shots you're after @lili2311 
@pavel-github looks like we need to rebase against master here too to resolve .travis.yaml merge conflict

**Comment by pavel-github** [comment]
@gitphill, yes. I fixed a typo on 'distribution' word :).

**Comment by pavel-github** [comment]
"bad" case:
snyk-gradle_without-fix

good case:
snyk-gradle_with-fix

File: .travis.yml [hunk 1]
```diff
@@ -18,15 +18,17 @@ before_script:
   - source "$HOME/.sdkman/bin/sdkman-init.sh"
   - echo sdkman_auto_answer=true > ~/.sdkman/etc/config
   - unset JAVA_HOME
+  - export GRADLE_OPTS=-Dorg.gradle.console=verbose
```
Comment (gitphill) [inline .travis.yml]: we don't need this since we override the env var?

Comment (gitphill) [reply .travis.yml]: or is this to test that we do override?

Comment (pavel-github) [reply .travis.yml]: yes, if we don't override, then the verbose console mode will break builds.

**Pull request merged**